// [[Reason: TODO: new is not correctly emitted.]] ////

//// [tests/cases/compiler/emitMethodCalledNew.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,13 +1,13 @@
 
 
 //// [emitMethodCalledNew.d.ts]
 export declare const a: {
-    "new"(x: number): number;
+    new(x: number): number;
 };
 export declare const b: {
-    "new"(x: number): number;
+    new(x: number): number;
 };
 export declare const c: {
-    "new"(x: number): number;
+    new(x: number): number;
 };
 //# sourceMappingURL=emitMethodCalledNew.d.ts.map
\ No newline at end of file
